arbeitszeit_flask/views/review_registered_consumptions_view.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+import flask
+
+from arbeitszeit.use_cases.review_registered_consumptions import (
+    ReviewRegisteredConsumptionsUseCase,
+)
+from arbeitszeit_flask import types
+from arbeitszeit_web.www.controllers.review_registered_consumptions_controller import (
+    InvalidRequest,
+    ReviewRegisteredConsumptionsController,
+)
+from arbeitszeit_web.www.presenters.review_registered_consumptions_presenter import (
+    ReviewRegisteredConsumptionsPresenter,
+)
+
+
+@dataclass
+class ReviewRegisteredConsumptionsView:
```

Comment:
Right, I forgot. Unfortunately the sql database implementation of the `PrivateConsumptionResult.where_provider_is_company` method is not implemented yet. That is why the integration test would fail. This method gets implemented in the open PR #927 though. I suggest to add the missing tests there.

Reply:
Alternatively we could xfail the view test here.

Reply:
Yes, thanks. See the last commit